src/data_loading.jl

```diff
@@ -74,4 +86,14 @@ function load(::MetabolicProfiles; timepoint_metadata = load(Metadata()))
     comm = CommunityProfile(Matrix(df[!, r"^FE"]), ms, MicrobiomeSample.(names(df, r"^FE")))
     set!(comm, timepoint_metadata; namecol=:etoh)
     return comm
+end
+
+function load(::Neuroimaging)
+    df = CSV.read(datafiles("exports", "raw_brain_measures.csv")) ## Have the raw_brain_measures.csv file in your exports directory.
+    subjs, tpts = split_subject_timepoint(df."Measure:volume")
+        df."Left-Thalamus" = df."Left-Thalamus" .+ df."Left-Thalamus-Proper"
+        df."Right-Thalamus" = df."Right-Thalamus" .+ df."Right-Thalamus-Proper"
+        df."IntraCranialVol" = df."EstimatedTotalIntraCranialVol" .+ df."IntraCranialVol"
+    df = hcat(DataFrame(:subject => subjs, :timepoint => tpts ), select(df, _reduced_braincols))
+    return sort(df, [:subject, :timepoint])
```

Comment:
```suggestion
    return sort!(df, [:subject, :timepoint])
```
No need to make a copy